README.md

```diff
@@ -1,154 +1,22 @@
-
-## Multi-line constructs
-
-Indent continuation lines by the standard 4 spaces. Always use trailing commas whenever legal (note that this differs between Python 2 and Python 3). Match the indentation of the closing bracket with the indentation of the start of the multi-line construct.
-
-```python
-# Tuple:
-my_tuple = (
-    foo,
-    bar,
-    baz,
-)
-
-# List:
-my_list = [
-    foo,
-    bar,
-    baz,
-]
-
-# Dictionary:
-my_dict = {
-    'foo': foo,
-    'bar': bar,
-    'baz': baz,
-}
-
-# Function signature:
-def my_func(
-    foo,
-    bar,
-    baz,
-):
-    pass
-
-# Function call:
-my_func(
-    foo,
-    bar,
-    baz,
-)
-
-# Function call with nested multiline construct:
-my_func(
-    foo,
-    (
-        bar,
-        baz,
-        qux,
-    ),
-)
-```
-
-When this syntax is not possible, indent by 8 spaces.
-
-```python
-# Multiple with statement:
-with \
-        context_manager_1(
-            foo,
-            bar,
-            baz,
-        ), \
-        context_manager_2(
-            foo,
-            bar,
-            baz,
-        ):
-    pass
-```
-
-Prefer to either put all elements on a single line, or have one element per line, except when grouping is especially semantically meaningful.
-
-```python
-# Good:
-my_tuple = (
-    foo,
-    bar,
-    baz,
-)
-
-# Good:
-my_tuple = (
-    foo, bar, baz,
-)
-
-# Usually bad:
-my_tuple = (
-    foo, bar,
-    baz,
-)
-```
-
-If a function takes two arguments, where the second argument is a collection, it is acceptable to use a multiline construct for only that second argument.
-
-```python
-my_func(foo, (
-    bar,
-    baz,
-    qux,
-))
-```
-
-As noted in PEP 8, prefer implicit line continuation inside brackets over explicit continuations with backslashes.
-
-```python
-# Good:
-from foo import (
-    bar, baz, qux,
-)
-
-# Bad:
-from foo import \
-    bar, baz, qux
-```
+We rely on [fourmat](https://github.com/4Catalyzer/fourmat) to automatically format and lint code according to our style guidelines. Below are additional guidelines that are not taken care of by `fourmat`.
 
 ## Data types
 
 Prefer immutable data types over their mutable equivalents. Specifically, prefer `tuple` and `frozenset` over `list` and `set` respectively when the relevant collections will not be mutated. However, it is acceptable to use `list` over `tuple` for collections that semantically constitute homogeneous lists rather than heterogeneous tuples, though `tuple` may still be better when the list is short.
 
 ## Import styles
 
-If a module defines a homogeneous collection of items (e.g. `models`, `views` or `layers`), prefer importing the module over the individual items from that module.
+If a module defines a homogeneous collection of items (e.g. `models`, `views` or `layers`), prefer importing the module over the individual items from that module.
```

Comment:
```suggestion
If a module defines a homogeneous collection of items (e.g. `models`, `views` or `layers`), prefer importing the module over the individual items from that module.
```

Reply:
huh?

Reply:
also this suggestion is probably wrong for flask-resty "resources"

Reply:
Hm, dunno what that space is about..probably some weirdness from importing then exporting from Notion.

Should we just remove this section?